### ISSUE TITLE
Some PromQL not work

### ISSUE THREAD

**Issue by axot** [VictoriaMetrics/VictoriaMetrics#150]
We are using a query, which is working in Prometheus but not with VictoriaMetrics.
The problem is in `!= inf != nan` part, if we can not use this syntax, any alternative we have?

```
container_spec_memory_limit_bytes{container_name!=""} != inf != nan
```

**Comment by valyala** [comment]
@axot , thanks for the bug report. The issue should be fixed in the commit 582fdc059a319a551ea319cdf56c295f57126014 . It will be available in the next VictoriaMetrics release.

**Comment by axot** [comment]
That’s impressive, so I will close this issue. Thank you.

**Comment by valyala** [comment]
@axot , could you verify whether the issue is fixed in [v1.25.2](https://github.com/VictoriaMetrics/VictoriaMetrics/releases)?

**Comment by axot** [comment]
@valyala I confirmed the issue was fixed in v1.25.2. Thank you.
